dev-packages/node-integration-tests/suites/sessions/crashed-session-aggregate/test.ts

```diff
@@ -4,16 +4,10 @@ afterEach(() => {
   cleanupChildProcesses();
 });
 
-test('should aggregate successful and crashed sessions', async () => {
-  let _done: undefined | (() => void);
-  const promise = new Promise<void>(resolve => {
-    _done = resolve;
-  });
-
-  const runner = createRunner(__dirname, 'server.ts')
+test('should aggregate successful and crashed sessions', done => {
```

Comment:
This and the other tests below were simply not doing anything at all, if you look through it in detail you see a bunch of things that can't work:

1. the path to the server.ts is wrong
2. The route paths are wrong